src/smc-webapp/jupyter/output-messages/image.tsx

```diff
@@ -43,18 +45,37 @@ export class Image extends Component<ImageProps, ImageState> {
     return this.props.type.split("/")[1].split("+")[0];
   };
 
+  img_click(): void {
+    this.setState(state => {
+      return { zoomed: !state.zoomed };
+    });
+  }
+
+  render_image(src, on_error?): Rendered {
+    const props = {
+      src,
+      width: this.props.width,
+      height: this.props.height,
+      onClick: this.img_click
+    };
+    if (this.props.width == null && this.props.height == null) {
+      const cursor = this.state.zoomed ? "zoom-out" : "zoom-in";
+      props["style"] = { cursor } as React.CSSProperties;
```

Comment:
Why are you writing props["style"] instead of props.style, and props['onError"] instead of props.onError.  It looks like you're writing **PYTHON** code....

Reply:
no idea, at least temporary my brain did output it that way